apps/src/sites/studio/pages/code-studio.js

```diff
@@ -9,24 +9,34 @@
 import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {getStore} from '@cdo/apps/code-studio/redux';
-import {setRtlFromDOM} from '@cdo/apps/code-studio/isRtlRedux';
+
+import hashEmail from '@cdo/apps/code-studio/hashEmail';
 import initSigninState from '@cdo/apps/code-studio/initSigninState';
+import {setRtlFromDOM} from '@cdo/apps/code-studio/isRtlRedux';
+import {
+  registerGetResult,
+  registerLevel,
+  onAnswerChanged,
+} from '@cdo/apps/code-studio/levels/codeStudioLevels';
+import {getStore} from '@cdo/apps/code-studio/redux';
 import initResponsive from '@cdo/apps/code-studio/responsive';
-import hashEmail from '@cdo/apps/code-studio/hashEmail';
+import {initHamburger} from '@cdo/apps/hamburger/hamburger.js';
 import GDPRDialog from '@cdo/apps/templates/GDPRDialog';
+// disable import/order rule to import consoleShim after setting store.
+// This might be safe to remove but needs investigation whether any behavior is changed by order.
+/* eslint-disable import/order*/
 import getScriptData from '@cdo/apps/util/getScriptData';
 
 const store = getStore();
 store.dispatch(setRtlFromDOM());
-
 // Shim window.console to be safe in IE
 require('@cdo/apps/code-studio/consoleShim')(window);
```

Comment:
This should be in a separate PR if we want to remove it.

Reply:
Yes, agreed. It would probably be cleanest to remove it in a prerequisite PR so this one could be applied cleanly, but I think taking care of it in a fast follow-up should be just fine.